data/websites.json

```diff
@@ -45,6 +50,11 @@
       "url": "https://cbs.gov.np/",
       "description": "Central Beuro of Statistics"
     },
+      {
```

Comment:
I don't think the JSON is properly formatter using prettier here, can you please fix it. 🙏